packages/botonic-react/src/components/message.jsx

```diff
@@ -217,45 +267,15 @@ export const Message = props => {
             blobWidth={getThemeProperty('message.bot.blobWidth')}
             blob={blob}
             style={{
-              border: `1px solid ${getThemeProperty(
-                'message.user.style.background',
-                getBgColor()
-              )}`,
               ...getMessageStyle(),
               ...style,
             }}
             {...otherProps}
           >
             <BlobText blob={blob}>{textChildren}</BlobText>
             {buttons}
-            {isFromUser() && blob && getBlobTick() && (
-              <div
-                style={{
-                  ...pointerStyles,
-                  borderLeftColor: getThemeProperty(
-                    'message.user.style.background',
-                    getBgColor()
-                  ),
-                  right: 0,
-                  borderRight: 0,
-                  marginRight: -pointerSize,
-                }}
-              />
-            )}
-            {isFromBot() && blob && getBlobTick() && (
-              <div
-                style={{
-                  ...pointerStyles,
-                  borderRightColor: getThemeProperty(
-                    'message.bot.style.background',
-                    getBgColor()
-                  ),
-                  left: 0,
-                  borderLeft: 0,
-                  marginLeft: -pointerSize + 1,
-                }}
-              />
-            )}
+            {blob && hasBlobTick() && getBlobTick(6)}
+            {blob && hasBlobTick() && getBlobTick(5)}
```

Comment:
If `hasBlobTick()` returns a `boolean`, you did the right thing here with the refactor of the method 👍